tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -1880,6 +1891,123 @@ tensorflow::Status ConvertReshape(OpConverterParams* params) {
   return tensorflow::Status::OK();
 }
 
+tensorflow::Status ConvertExpandDims(OpConverterParams* params) {
+  const auto& inputs = params->inputs;
+  const auto& node_def = params->node_def;
+  if (inputs.size() != 2) {
+    return tensorflow::errors::InvalidArgument(
+        "Two inputs expected for ExpandDims, at ", node_def.name());
+  }
+  if (!inputs.at(1).is_weights() ) {
```

Comment:
Hmm, I guess so. I didn't restrict it to tensors because I noticed some ExpandDims and Squeeze ops in my ConvSeq2Seq model had weight inputs (possibly const fold bug) and PrepareTensorForShape will work for either type of input. But allowing the op to be converted in those cases could introduce more problems elsewhere.